remote_attestation/rust/Cargo.toml

```diff
@@ -5,15 +5,17 @@ authors = ["Ivan Petrov <ivanpetrov@google.com>"]
 edition = "2021"
 license = "Apache-2.0"
 
+[features]
+default = ["alloc"]
+std = ["anyhow/std", "prost/std"]
+alloc = []
+
 [dependencies]
-anyhow = "*"
-bincode = "*"
+anyhow = { version = "*", default-features = false }
+bytes = { version = "*", default-features = false }
 log = "*"
-prost = "*"
+prost = { version = "*", default-features = false, features = ["prost-derive"] }
```

Comment:
Since we are not restricting versions, is it possible that the future versions of our dependencies will become `std` instead of `no_std`?

Reply:
Yes, it is possible. I am not sure whether there is a good way to detect that automatically.

Reply:
Wouldn't this crate fail to compile in that case (at least with the defaul features, i.e. alloc instead of std)?

Reply:
Drive-by: the best way I've found to check `no_std` compatibility is to have a [CI step](https://github.com/google/coset/blob/main/.github/workflows/ci.yml#L90) that cross-compiles for a target with no `std` available.

Reply:
Thanks. At the moment the build requires a target OS because that is required by `ring`. We should be able to build it for a UEFI target once https://github.com/briansmith/ring/pull/1406 has been merged.